ipdb/__main__.py

```
@@ -23,20 +25,65 @@
+            profile_dir = ProfileDir.find_profile_dir_by_name(
+                ipython_dir=ipython_dir,
+                name=ipython_profile,
+            )
+        except ProfileDirError:
+            # Profile doesn't exist, try to create it
+            try:
+                profile_dir = ProfileDir.create_profile_dir_by_name(
+                    ipython_dir=ipython_dir,
+                    name=ipython_profile,
+                )
+            except (ProfileDirError, Exception):
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)